deps/verifier/src/snp/mod.rs

```diff
@@ -34,43 +35,43 @@ const LOADER_SPL_OID: Oid<'static> = oid!(1.3.6 .1 .4 .1 .3704 .1 .3 .1);
 
 #[derive(Debug)]
 pub struct Snp {
-    vendor_certs: VendorCertificates,
+    vendor_certs: Chain,
 }
 
-pub(crate) fn load_milan_cert_chain() -> &'static Result<VendorCertificates> {
-    static MILAN_CERT_CHAIN: OnceLock<Result<VendorCertificates>> = OnceLock::new();
-    MILAN_CERT_CHAIN.get_or_init(|| {
-        let certs = X509::stack_from_pem(include_bytes!("milan_ask_ark_asvk.pem"))?;
-        if certs.len() != 3 {
-            bail!("Malformed Milan ASK/ARK/ASVK");
+fn read_certs_from_file() -> Vec<X509> {
+    match X509::stack_from_pem(include_bytes!("milan_ask_ark_asvk.pem")) {
+        Result::Ok(certs) => {
+            if certs.len() != 3 {
+                panic!("Malformed Milan ASK/ARK/ASVK");
+            }
+            return certs;
         }
+        Result::Err(err) => panic!("Error reading certificates file: {err}"),
```

Comment:
I think initially there was an ask to avoid panicking if the parsing of the bundled X509 failed. I'm fine with this, since this is very const-y and unlikely to fail.

However, we're not reading a file here, the bytes are embedded in the binary.

Reply:
Renamed the file to more accurately describe what is happening.